pyfr/solvers/navstokes/kernels/avis.mako

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+<%inherit file='base'/>
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+<%
+    import math
```

Comment:
I have tried that, but it does not work. I have also tested following simple script, but the last line gives error.

from mako.template import Template
print(Template("<% import math %>${math.pi}").render())
print(Template("<%namespace module='math' name='math'/>${math.pi}").render())